shared-workers.md

```diff
@@ -27,11 +27,28 @@ There has been increasing [developer](https://github.com/GoogleChromeLabs/privac
 
 ## [Maintaining a Session](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support/issues/124)
 
-A developer embeds chat.com on two of their sites site-a.com and site-b.com. chat.com uses Shared Workers to maintain a user session.
+Some websites allow users to chat with the site's customer service.
+For example, the US internet provider Comcast splits parts of its services across business.comcast.com, xfinity.com, and more.
+In some cases, pieces of a user's natural journey while interacting with the customer service agent lead them to different but associated sites.
+This could be implemented by putting a session ID into a cookie, and that would continue to work while cookies are partitioned if the user grants storage access.
+However, it's more natural and common for sites to use a SharedWorker to manage the chat session.
+This proposal allows that to work after the same use permission grant.
+
+A website, company-chat.example, offers a way to maintain active chat session between any sites the embed it using SharedWorkers.
+Before storage partitioning this was possible, but after storage partitioning when company-chat.example is embedded on different sites (company-site1.example or company-site2.example) and instantiates a SharedWorker it would no longer be shared.
+By prompting the user for permission via `document.requestStorageAccess({SharedWorker: true})` and then instantiating the SharedWorker via the returned handle the worker can be shared across partitioned third-party contexts.
```

Comment:
Thanks! I think the introduction matches https://w3ctag.github.io/explainer-explainer/#end-user-need now.

The second paragraph could flow better though: you've already introduced the idea of an active chat session, so you could start with something like, "To implement this, the company embeds a helper site, e.g. `company-chat.example`, into all of its branded sites, and `company-chat.example` uses a `SharedWorker` to communicate between its multiple embedded instances."